app/components/big-text.js

```diff
@@ -23,7 +23,12 @@ export default Component.extend({
     }
   }),
   cleanText: computed('text', function(){
-    const text = this.get('text') || '';
+    let text = this.get('text') || '';
+
+    if ((typeof text === 'object' && 'toString' in text || typeof text === 'number')){
```

Comment:
possible to user `Ember.typeOf()` here instead of rolling a manual type detection? http://emberjs.com/api/#method_typeOf